scss/widgets/post.scss

```diff
@@ -120,6 +120,42 @@
     margin: 0 0 ($spacer * 0.5) 0;
     line-height: 1;
   }
+
+  .dc-topic-info {
+    margin-bottom: 0.5rem;
+
+    .dc-topic-category,
+    .dc-topic-tag {
+      color: $gray-800 !important;
```

Comment:
As this is a sort of new widget the `!important` is really need it?

Reply:
Yeah, actually it is not needed